test/verify/check-docker-storage

```diff
@@ -38,7 +38,7 @@ import unittest
 #
 def can_manage(machine, vg):
     # Atomic is all set.
-    if machine.image in [ "rhel-atomic", "fedora-atomic", "continuous-atomic" ]:
+    if machine.atomic_image:
```

Comment:
This has not always been the case for all Atomics. I would suggest leave the original check in place, so we can add/remove them individually.